lib/cfd_trading/cfd_offer.dart

```diff
@@ -43,9 +41,8 @@ class _CfdOfferState extends State<CfdOffer> {
         expiry: DateTime.now(),
         liquidationPrice: 13104,
         openPrice: 19100,
-        pl: Amount(Random().nextInt(20000) + -10000),
         quantity: 100,
-        estimatedFees: Amount(-4));
+        estimatedOpeningFees: Amount(-4));
```

Comment:
❓ Could we just remove the opening fee for the tournament?

Reply:
It was removed as far as I could see, sry should have squashed some commits :)